src/main/scala/mesosphere/marathon/core/task/tracker/InstanceStateOpProcessor.scala

```diff
@@ -14,4 +18,14 @@ import scala.concurrent.Future
 trait InstanceStateOpProcessor {
   /** Process an InstanceUpdateOperation and propagate its result. */
   def process(stateOp: InstanceUpdateOperation): Future[InstanceUpdateEffect]
```

Comment:
can we make this private? Or is this still called from outside?

Reply:
Unfortunately there is still one call deep from withing the launch queue :/.